stdlib/2/__builtin__.pyi

```diff
@@ -1440,12 +1440,12 @@ Ellipsis: ellipsis
 
 if sys.version_info < (3,):
     # TODO: buffer support is incomplete; e.g. some_string.startswith(some_buffer) doesn't type check.
-    _AnyBuffer = TypeVar('_AnyBuffer', str, unicode, bytearray, buffer)
+    _ABT = TypeVar('_ABT', str, unicode, bytearray, buffer)
 
     class buffer(Sized):
-        def __init__(self, object: _AnyBuffer, offset: int = ..., size: int = ...) -> None: ...
-        def __add__(self, other: _AnyBuffer) -> str: ...
-        def __cmp__(self, other: _AnyBuffer) -> bool: ...
+        def __init__(self, object: _ABT, offset: int = ..., size: int = ...) -> None: ...
+        def __add__(self, other: _ABT) -> str: ...
+        def __cmp__(self, other: _ABT) -> bool: ...
```

Comment:
Should these be just unions?

Reply:
Good point. I changed it to a union and changed the name back to be more expressive.